diff --git a/contracts/RedeemableERC20Pool.sol b/contracts/RedeemableERC20Pool.sol
index 533c62a..f64c7ec 100644
--- a/contracts/RedeemableERC20Pool.sol
+++ b/contracts/RedeemableERC20Pool.sol
@@ -19,7 +19,7 @@ import { IBPool } from "./configurable-rights-pool/contracts/IBFactory.sol";
 import { BPool } from "./configurable-rights-pool/contracts/test/BPool.sol";
 import { RightsManager } from "./configurable-rights-pool/libraries/RightsManager.sol";
 import { BalancerConstants } from "./configurable-rights-pool/libraries/BalancerConstants.sol";
-import { ConfigurableRightsPool } from "./configurable-rights-pool/contracts/ConfigurableRightsPool.sol";
+import { ConfigurableRightsPool, PoolParams } from "./configurable-rights-pool/contracts/ConfigurableRightsPool.sol";
 import { CRPFactory } from "./configurable-rights-pool/contracts/CRPFactory.sol";
 import { BFactory } from "./configurable-rights-pool/contracts/test/BFactory.sol";
 
@@ -173,7 +173,7 @@ contract RedeemableERC20Pool is Ownable, Initable, BlockBlockable {
 
         return poolConfig_.crpFactory.newCrp(
             address(poolConfig_.balancerFactory),
-            ConfigurableRightsPool.PoolParams(
+            PoolParams(
                 "R20P",
                 "RedeemableERC20Pool",
                 poolAddresses_,
