diff --git a/contracts/RedeemableERC20Pool.sol b/contracts/RedeemableERC20Pool.sol
index fedbe77..7c75045 100644
--- a/contracts/RedeemableERC20Pool.sol
+++ b/contracts/RedeemableERC20Pool.sol
@@ -13,7 +13,7 @@ import { IBPool } from "./configurable-rights-pool/contracts/IBFactory.sol";
 import { BPool } from "./configurable-rights-pool/contracts/test/BPool.sol";
 import { RightsManager } from "./configurable-rights-pool/libraries/RightsManager.sol";
 import { BalancerConstants } from "./configurable-rights-pool/libraries/BalancerConstants.sol";
-import { ConfigurableRightsPool } from "./configurable-rights-pool/contracts/ConfigurableRightsPool.sol";
+import { ConfigurableRightsPool, PoolParams } from "./configurable-rights-pool/contracts/ConfigurableRightsPool.sol";
 import { CRPFactory } from "./configurable-rights-pool/contracts/CRPFactory.sol";
 import { BFactory } from "./configurable-rights-pool/contracts/test/BFactory.sol";
 
@@ -132,7 +132,7 @@ contract RedeemableERC20Pool is Ownable, Phased {
 
         crp = config_.crpFactory.newCrp(
             address(config_.balancerFactory),
-            ConfigurableRightsPool.PoolParams(
+            PoolParams(
                 "R20P",
                 "RedeemableERC20Pool",
                 poolAddresses_,
