diff --git a/contracts/RedeemableERC20Pool.sol b/contracts/RedeemableERC20Pool.sol
index 354a491..efb13c3 100644
--- a/contracts/RedeemableERC20Pool.sol
+++ b/contracts/RedeemableERC20Pool.sol
@@ -18,7 +18,7 @@ import {
     BalancerConstants
 } from "./configurable-rights-pool/libraries/BalancerConstants.sol";
 import {
-    ConfigurableRightsPool
+    ConfigurableRightsPool, PoolParams
 } from "./configurable-rights-pool/contracts/ConfigurableRightsPool.sol";
 import {
     CRPFactory
@@ -170,7 +170,7 @@ contract RedeemableERC20Pool is Ownable, Phased {
 
         ConfigurableRightsPool crp_ = config_.crpFactory.newCrp(
             address(config_.balancerFactory),
-            ConfigurableRightsPool.PoolParams(
+            PoolParams(
                 "R20P",
                 "RedeemableERC20Pool",
                 poolAddresses_,
