Submodule contracts/configurable-rights-pool contains modified content
diff --git a/contracts/configurable-rights-pool/contracts/CRPFactory.sol b/contracts/configurable-rights-pool/contracts/CRPFactory.sol
index 52ef23b..532f5a1 100644
--- a/contracts/configurable-rights-pool/contracts/CRPFactory.sol
+++ b/contracts/configurable-rights-pool/contracts/CRPFactory.sol
@@ -49,7 +49,7 @@ contract CRPFactory {
      */
     function newCrp(
         address factoryAddress,
-        ConfigurableRightsPool.PoolParams calldata poolParams,
+        PoolParams calldata poolParams,
         RightsManager.Rights calldata rights
     )
         external
diff --git a/contracts/configurable-rights-pool/contracts/ConfigurableRightsPool.sol b/contracts/configurable-rights-pool/contracts/ConfigurableRightsPool.sol
index e3df626..e98f5c7 100644
--- a/contracts/configurable-rights-pool/contracts/ConfigurableRightsPool.sol
+++ b/contracts/configurable-rights-pool/contracts/ConfigurableRightsPool.sol
@@ -18,6 +18,17 @@ import { RightsManager } from "../libraries/RightsManager.sol";
 import "../libraries/SmartPoolManager.sol";
 import "../libraries/SafeApprove.sol";
 
+struct PoolParams {
+    // Balancer Pool Token (representing shares of the pool)
+    string poolTokenSymbol;
+    string poolTokenName;
+    // Tokens inside the Pool
+    address[] constituentTokens;
+    uint[] tokenBalances;
+    uint[] tokenWeights;
+    uint swapFee;
+}
+
 // Contracts
 
 /**
@@ -44,16 +55,7 @@ contract ConfigurableRightsPool is PCToken, BalancerOwnable, BalancerReentrancyG
 
     // Type declarations
 
-    struct PoolParams {
-        // Balancer Pool Token (representing shares of the pool)
-        string poolTokenSymbol;
-        string poolTokenName;
-        // Tokens inside the Pool
-        address[] constituentTokens;
-        uint[] tokenBalances;
-        uint[] tokenWeights;
-        uint swapFee;
-    }
+
 
     // State variables
diff --git a/contracts/configurable-rights-pool/contracts/templates/ESPFactory.sol b/contracts/configurable-rights-pool/contracts/templates/ESPFactory.sol
index 6fe5b44..8a3fa1e 100644
--- a/contracts/configurable-rights-pool/contracts/templates/ESPFactory.sol
+++ b/contracts/configurable-rights-pool/contracts/templates/ESPFactory.sol
@@ -48,7 +48,7 @@ contract ESPFactory {
      */
     function newEsp(
         address factoryAddress,
-        ConfigurableRightsPool.PoolParams calldata poolParams,
+        PoolParams calldata poolParams,
         RightsManager.Rights calldata rights
     )
         external
diff --git a/contracts/configurable-rights-pool/contracts/templates/ElasticSupplyPool.sol b/contracts/configurable-rights-pool/contracts/templates/ElasticSupplyPool.sol
index 9d58e72..c183f1e 100644
--- a/contracts/configurable-rights-pool/contracts/templates/ElasticSupplyPool.sol
+++ b/contracts/configurable-rights-pool/contracts/templates/ElasticSupplyPool.sol
@@ -88,7 +88,7 @@ contract ElasticSupplyPool is ConfigurableRightsPool {
      */
     constructor(
         address factoryAddress,
-        ConfigurableRightsPool.PoolParams memory poolParams,
+        PoolParams memory poolParams,
         RightsManager.Rights memory rightsParams
     )
         // solhint-disable-next-line visibility-modifier-order
