diff --git a/contracts/pool/RedeemableERC20Pool.sol b/contracts/pool/RedeemableERC20Pool.sol
index 7334229..89da700 100644
--- a/contracts/pool/RedeemableERC20Pool.sol
+++ b/contracts/pool/RedeemableERC20Pool.sol
@@ -24,7 +24,8 @@ import {
 } from
 "@beehiveinnovation/configurable-rights-pool/libraries/BalancerConstants.sol";
 import {
-    ConfigurableRightsPool
+    ConfigurableRightsPool,
+    PoolParams
 // solhint-disable-next-line max-line-length
 } from "@beehiveinnovation/configurable-rights-pool/contracts/ConfigurableRightsPool.sol";
 import {
@@ -211,7 +212,7 @@ contract RedeemableERC20Pool is Ownable, Phased {
 
         ConfigurableRightsPool crp_ = config_.crpFactory.newCrp(
             address(config_.balancerFactory),
-            ConfigurableRightsPool.PoolParams(
+            PoolParams(
                 "R20P",
                 "RedeemableERC20Pool",
                 poolAddresses_,
